modules/cache/main.tf

```diff
@@ -25,6 +25,8 @@ resource "random_string" "s3_suffix" {
   special = false
 }
 
+# ok, we can go without access logging as it is a cache bucket only
```

Comment:
This is certainly OK for me, and can be made advanced later. But see here a PR on one of my other repos where optional logging is added

https://github.com/philips-labs/terraform-aws-github-runner/pull/2430/files#diff-3d07fd4aceafd6afc7a21efd7392168bb00a3bb8f1a3c5099aa176b8bd8ecb96R65

Reply:
Good point! We should consider this.

Reply:
Logging support added